decentraland/unity-explorer
# fix: temporarily remove the realtime reflection probe

## What does this PR change?

We are temporarily removing the realtime reflection probe while we create a better more optimal solution.

## How to test?

A quick smoke test verify build runs normally, noticable difference will be in the reflections of objects.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
